maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/PluginReport.java

```diff
@@ -526,8 +524,7 @@ public void renderBody()
                 tableHeader( new String[]{ goalColumnName, descriptionColumnName } );
             }
 
-            List<MojoDescriptor> mojos = new ArrayList<>();
-            mojos.addAll( pluginDescriptor.getMojos() );
+            List<MojoDescriptor> mojos = new ArrayList<>( pluginDescriptor.getMojos() );
```

Comment:
this works, but the existing code didn't make me look up the API docs to be sure of that. Clearer to use addAll.